txbits/app/views/exchange/trade.scala.html

```diff
@@ -167,6 +174,7 @@
                 </div>
             </div>
             <button type="submit" id="ask-submit" class="ask btn btn-danger btn-block">@{Messages("terminology.ask")} {{base}}</button>
+            {{#if one_way}}* @{Messages("help.one_way")}{{/if}}
```

Comment:
I didn't put it before the button because I felt like it got in the way too much